setup.cfg

```diff
@@ -34,7 +34,7 @@ setup_requires = setuptools_scm
 include_package_data = True
 install_requires =
     numpy>=1.23
-    astropy>=4.3
+    astropy>=6.0
     extension-helpers>=0.1
     packaging>=0.19
     scipy>=1.7
```

Comment:
Can we also update the ginga requirement to >=5.0.1, given the recent work by @ejeschke?

Reply:
Done!